tests/Dfe.PlanTech.Web.SeedTestData/Program.cs

```diff
@@ -0,0 +1,57 @@
+using Dfe.PlanTech.Domain.Persistence.Models;
+using Dfe.PlanTech.Infrastructure.Data;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+
+namespace Dfe.PlanTech.Web.SeedTestData;
+
+public static class Program
+{
+    private static ServiceProvider CreateServiceProvider(IConfiguration? configuration = null)
+    {
+        configuration ??= CreateConfiguration();
+
+        var services = new ServiceCollection();
+
+        services.AddDbContext<CmsDbContext>(opts =>
+        {
+            opts.UseSqlServer(configuration.GetConnectionString("Database"));
+            opts.EnableSensitiveDataLogging();
+        });
+
+        services.AddLogging(opts =>
+        {
+            opts.AddConsole();
+            opts.SetMinimumLevel(LogLevel.Warning);
+        });
+
+        services.AddSingleton(new ContentfulOptions(true));
+
+        services.AddSingleton<SeedData>();
+
+        return services.BuildServiceProvider();
+    }
+
+    private static IConfiguration CreateConfiguration()
+    {
+        var configurationBuilder = new ConfigurationBuilder();
+        configurationBuilder.AddUserSecrets<ForLoadingUserSecrets>();
```

Comment:
Do we need a class just for this? Any reason we can't either:
1. Use one of the other classes (e.g. `ContentGenerator`)
2. Just use the assembly? E.g. `configurationBuilder.AddUserSecrets(Assembly. GetExecutingAssembly())`

Reply:
Great shout, done